frontend/frontend-filters.php

```diff
@@ -161,42 +166,146 @@ public function widget_display_callback( $instance ) {
 	public function sidebars_widgets( $sidebars_widgets ) {
 		global $wp_registered_widgets;
 
-		if ( empty( $wp_registered_widgets ) ) {
-			return $sidebars_widgets;
+		$existing_sidebars_widgets = $this->init_sidebars_widgets( $sidebars_widgets, $wp_registered_widgets );
+		if ( $existing_sidebars_widgets ) {
+			return $existing_sidebars_widgets;
 		}
```

Comment:
Should this be different than https://github.com/polylang/polylang-pro/blob/bcd43b2ebde21d6d8f34483e2cd2ff2ab4651887/modules/block-editor/widget-editor-language-attribute.php#L79 ?

Reply:
Not really, it's an omission, as this is the part that is used to exit the method.